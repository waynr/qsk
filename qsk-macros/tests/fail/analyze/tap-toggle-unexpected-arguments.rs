use qsk_macros::layer;

fn main() {
    layer!(
        ModLayer[Meow]: {
            F -> TapToggle(Navigation, F, MEOW),
        },
    );
}
