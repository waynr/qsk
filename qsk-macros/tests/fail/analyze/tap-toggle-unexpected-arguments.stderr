error: unexpected argument
 --> tests/fail/analyze/tap-toggle-unexpected-arguments.rs:6:43
  |
6 |             F -> TapToggle(Navigation, F, MEOW),
  |                                           ^^^^
